frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-row.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { ArrowCircleDownIcon } from '@patternfly/react-icons';
+import { LoadingInline } from '@console/internal/components/utils';
+
+const prefixedId = (idPrefix, id) => {
+  return idPrefix && id ? `${idPrefix}-${id}` : null;
+};
+
+const BucketsRowTitle = ({ count, title, objectsCount }) => (
+  <div className="co-buckets-card__row-title">
+    <div>{count != null ? `${count} ${title}` : title}</div>
+    <div className="co-buckets-card__row-subtitle">{`${objectsCount} Objects`}</div>
```

Comment:
```suggestion
    <div className="co-buckets-card__row-subtitle">{objectsCount} Objects}</div>
```

Consider using `pluralize`